main.go

```diff
@@ -118,15 +118,26 @@ func destroyDNS(c *conf, hc *http.Client) error {
 	req.Header.Add("Accept", "application/json")
 	req.Header.Add("Authorization", fmt.Sprintf("Basic %s", c.konsAPIKey))
 
-	_, err = hc.Do(req)
+	response, err := hc.Do(req)
 	if err != nil {
 		return err
 	}
+	defer response.Body.Close()
+
+	if response.StatusCode != 200 {
+		// if status is not 200, log it, but do not consider it as a service failure
```

Comment:
I don't see how much better we can make this without asking for a constructor change that gives us a 1-hit API call for our use case.  Worth doing that?